app/api/endpoints.py

```diff
@@ -57,6 +62,9 @@ def get_domains():
     sites_available = []
     sites_enabled = []
 
+    if not os.path.exists(config_path):
+        errorMessage = 'The config folder "{}" does not exists.'.format(config_path)
+        return flask.render_template('domains.html', errorMessage=errorMessage, sites_available=(), sites_enabled=()), 200
```

Comment:
Classic case of Java developer. Lower CamelCase is not common in Python. error_message would be nice! :)

Reply:
I would also like to have the string formatting with f-string syntax. Then it will be consistent in the code.
```python
error_message = f'The config folder "{config_path}" don't exists.'
```